hca/util/__init__.py

```diff
@@ -115,9 +115,19 @@ def special_cli_command(self, required_argument, optional_argument=""):
 
 
 class RetryPolicy(retry.Retry):
-    def __init__(self, retry_after_status_codes={301}, *args, **kwargs):
+    def __init__(self,
+                 retry_after_status_codes={301},
+                 read=10,
+                 status=10,
+                 backoff_factor=0.1,
+                 status_forcelist=frozenset(set(range(500, 512))),
```

Comment:
let's preserve the original set for now: `{500, 502, 503, 504}`

Reply:
Sounds good.  Will close this until I have something to test with.